pkg/account/deployer/deployer.go

```diff
@@ -185,12 +201,14 @@ func (d *AccountDeployer) updateGroupPolicyBindings(ctx context.Context, group a
 	if remoteGroupId == "" {
 		return fmt.Errorf("unable to determine UUID for group %q", group.Name)
 	}
-	remoteIds, err := d.getAccountPolicyRefs(group)
+	accPolicyUuids, err := d.getAccountPolicyRefs(group)
 	if err != nil {
 		return fmt.Errorf("failed to fetch policies for group %q: %w", group.Name, err)
 	}
 
-	if err = d.accountManagementClient.updateAccountPolicyBindings(ctx, remoteGroupId, remoteIds); err != nil {
+	log.Debug("Updating account level policy bindings for group with ID %q", remoteGroupId)
+	log.Debug("  Bind group %q --> %v", remoteGroupId, accPolicyUuids)
```

Comment:
nit: We've just started to get rid of indented log lines. If this is related to the log above, I'd rather we print it as one log entry (possibly with a newline and indent), rather than only "visually" making two log entries related by indenting the next line